internal/host/common.go

```diff
@@ -30,7 +30,7 @@ const (
 	statusInfoDiscovering                    = "Waiting for host to send hardware details"
 	statusInfoInsufficientHardware           = "Host does not meet the minimum hardware requirements"
```

Comment:
You might want to do the same for statusInfoInsufficientHardware and statusInfoPendingForInput

Reply:
added separate ticket: https://issues.redhat.com/browse/MGMT-2831